### ISSUE TITLE
Build failed on OS X 10.10.1 for 2.3.0-preview1 & 2.3.0-preview-2

### ISSUE THREAD

**Issue by olivierlacan** [rbenv/ruby-build#848]
On a clean install of OS X 10.11.1 with Homebrew installed (and no issues reported by `brew doctor`): 

```
generating x86_64-darwin15-fake.rb
./tool/generic_erb.rb:2:in `require': cannot load such file -- erb (LoadError)
	from ./tool/generic_erb.rb:2:in `<main>'
clang: error: unable to execute command: Broken pipe: 13
clang: error: clang frontend command failed due to signal (use -v to see invocation)
Apple LLVM version 7.0.0 (clang-700.1.76)
Target: x86_64-apple-darwin15.0.0
Thread model: posix
clang: note: diagnostic msg: PLEASE submit a bug report to http://developer.apple.com/bugreporter/ and include the crash backtrace, preprocessed source, and associated run script.
clang: note: diagnostic msg:
```

Build log here for preview1: https://gist.github.com/olivierlacan/2d719579f1f36430399d

**Comment by jeremy** [comment]
Does it affect other Ruby builds, or just these two?

**Comment by jeremy** [comment]
Also reported here: https://github.com/rbenv/ruby-build/pull/824#commitcomment-14378878

**Comment by olivierlacan** [comment]
@jeremy Will test.

**Comment by olivierlacan** [comment]
@jeremy Actually did test. No it affects only these two. I've successfully installed a 1.9, 2.0, 2.1 and 2.2 MRI on the same machine.

**Comment by hsbt** [comment]
I got same error from another user: https://gist.github.com/kurotaky/726a442fc8812fb2e142

It's defects of ruby package.

what is version of your default ruby(which -a ruby)?

**Comment by olivierlacan** [comment]
@hsbt It's 2.0.0p645. It ships with OS X 10.11.

```
$ which -a ruby
/Users/olivierlacan/.rbenv/shims/ruby
/usr/bin/ruby
$ /usr/bin/ruby -v
ruby 2.0.0p645 (2015-04-13 revision 50299) [universal.x86_64-darwin15]
```

Do you suggest upgrading the system version?

**Comment by hsbt** [comment]
> Do you suggest upgrading the system version?

@olivierlacan No, system version is not related this issue. I can't reproduce this issue with OSX 10.11 and my enviroment.

Ruby core team(includes me) have to fix before 2.3.0 release. I addressed this issue to our tracker https://bugs.ruby-lang.org/issues/11807

**Comment by olivierlacan** [comment]
@hsbt Appreciate your help on this. I'll follow up and subscribe to the ruby-lang issue.

**Comment by hsbt** [comment]
see https://github.com/rbenv/ruby-build/issues/833#issuecomment-164148732

**Comment by olivierlacan** [comment]
@hsbt I installed both preview1 and preview2 from 2.1.6p336. So this is unlikely to be the cause.

**Comment by olivierlacan** [comment]
@hsbt My bad, I just noticed I didn't set a global Ruby version. I'm sorry. We should probably file a radar bug report with Apple to let them know that they need to move off of ruby 2.0.0p645, no?

**Comment by hsbt** [comment]
It causes rbenv regression addressed  https://github.com/rbenv/rbenv/commit/dc23ef10b6e8c07f0098a8fc19d6100c2dcee473 . so you need not report Apple :)

Please update your rbenv.

**Comment by olivierlacan** [comment]
@hsbt Oh okay, good to know. Thanks :-)

**Comment by olivierlacan** [comment]
This workaround suggested by @hsbt fixed my issue:

```
rbenv install 2.2.3
rbenv global 2.2.3
```

That said I'm going to leave this open until I can verify the regression fix from https://github.com/rbenv/rbenv/commit/dc23ef10b6e8c07f0098a8fc19d6100c2dcee473.

**Comment by olivierlacan** [comment]
Fix from https://github.com/rbenv/rbenv/commit/dc23ef10b6e8c07f0098a8fc19d6100c2dcee473 verified with `rbenv global system` set to `ruby 2.0.0p645 (2015-04-13 revision 50299) [universal.x86_64-darwin15]` which is the system default on OS 10.11.1.

Closing.

**Comment by hsbt** [comment]
Thank you for your feedback. I'm happy to fix this issue before Ruby 2.3.0 release.
